test/controllers/web-payments/handle-auth-response.controller.test.js

```diff
@@ -274,23 +278,13 @@ describe('The web payments handle auth response controller', () => {
       },
       deleteSessionVariable: sinon.spy()
     }
-    const systemErrorObj = {
-      viewName: 'SYSTEM_ERROR',
-      returnUrl: '/return/3',
-      analytics: {
-        analyticsId: 'test-1234',
-        type: 'test',
-        paymentProvider: 'sandbox',
-        path: '/handle-payment-response/3/error',
-        amount: '4.99',
-        testingVariant: 'original'
-      }
-    }
+
     requireHandleAuthResponseController(mockCharge, mockNormaliseCharge, mockCookies)(req, res)
     expect(mockCookies.deleteSessionVariable.calledWith(req, `ch_${chargeId}.webPaymentAuthResponse`)).to.be.ok // eslint-disable-line
-    expect(res.render.calledWith('errors/system-error', systemErrorObj)).to.be.true // eslint-disable-line
+    expect(res.render.calledWith('errors/incorrect-state/auth-failure')).to.be.true // eslint-disable-line
```

Comment:
would it be worth checking on the options passed when render is called, as this test was previously? In particular, we'd want to ensure the `returnUrl` is there as expected because the view needs this.

Reply:
👍